src/screens/flows/WalletConnect/screens/Flows/components/RequestCall/index.tsx

```diff
@@ -1,20 +1,127 @@
-import React from 'react';
-import { Text } from 'react-native';
+import { t } from 'i18next';
+import React, { useEffect, useState } from 'react';
+import { ActivityIndicator, Image, Linking, Text, View } from 'react-native';
+import { useSelector } from 'react-redux';
 
-import { WallectConnectFlowsData } from '@/interfaces/walletConnect';
+import { getTokenPrices } from '@/constants/assets';
+import { TRUST_AND_SECURITY_URL } from '@/constants/general';
+import { FontStyles } from '@/constants/theme';
+import { Nullable } from '@/interfaces/general';
+import { State } from '@/interfaces/redux';
+import {
+  WallectConnectFlowsData,
+  WCWhiteListItem,
+} from '@/interfaces/walletConnect';
+import { getUsdAvailableAmount } from '@/screens/flows/Send/utils';
+import { addSpacesAndCapitalize } from '@/utils/strings';
+import { getDabNfts } from '@/utils/walletConnect';
```

Comment:
Add a TODO here to use the `/services/DAB.ts` function after 0.2.0 merge, all dab functions are there